src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -28,73 +29,69 @@
 
 class MissingOrgMemberSerializer(Serializer):
     def serialize(self, obj, attrs, user, **kwargs):
-        return {"email": obj.email, "externalId": obj.external_id, "commitCount": obj.commit_count}
+        return {"email": obj.email, "externalId": obj.external_id, "commitCount": obj.commit__count}
 
 
 class MissingMembersPermission(OrganizationPermission):
     scope_map = {"GET": ["org:write"]}
 
 
-@region_silo_endpoint
-class OrganizationMissingMembersEndpoint(OrganizationEndpoint):
-    publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
-    }
-    permission_classes = (MissingMembersPermission,)
-
-    def _get_missing_members(
-        self, organization: Organization, provider: str, integration_ids: Sequence[int]
-    ) -> QuerySet[CommitAuthor]:
-        member_emails = set(
-            organization.member_set.exclude(email=None).values_list("email", flat=True)
-        )
-        member_emails.update(
-            set(
-                organization.member_set.exclude(user_email=None).values_list(
-                    "user_email", flat=True
-                )
-            )
-        )
-        nonmember_authors = CommitAuthor.objects.filter(organization_id=organization.id).exclude(
-            Q(email__in=member_emails) | Q(external_id=None)
+def _get_missing_organization_members(
+    organization: Organization, provider: str, integration_ids: Sequence[int]
+) -> QuerySet[WithAnnotations[CommitAuthor]]:
+    member_emails = set(organization.member_set.exclude(email=None).values_list("email", flat=True))
+    member_emails.update(
```

Comment:
Huh.
Do you need to do this in 2 steps?
maybe instead of an update you could splat both of these into the same set construction? 🤷

Reply:
oh i can do set union with `set(A) | set(B)`